tests/drivers/bc12/src/pd_mode.c

```diff
@@ -19,15 +19,15 @@
 
 #include <zephyr/logging/log.h>
 
-#define LOG_MODULE_NAME test_bc12
+#define LOG_MODULE_NAME test_bc12_pd
```

Comment:
Should this be in the previous commit? Same question for some of the other name changes below.